data/mods/MindOverMatter/monsters/feral_psychics.json

```
@@ -330,6 +330,224 @@
       "ALL_SEEING"
     ]
   },
+  {
+    "id": "mon_feral_human_photokin",
+    "type": "MONSTER",
+    "name": "feral radical",
+    "description": "This feral's skin has some sort of radiance. Their eyes are bright with light and there's a slight shimmer around their form.",
+    "default_faction": "zombie",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "color": "yellow",
+    "symbol": "@",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "bash", "amount": 4 } ],
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology" ],
+    "dodge": 3,
+    "harvest": "human",
+    "dissect": "dissect_human_sample_single",
+    "vision_day": 30,
+    "vision_night": 3,
+    "path_settings": { "max_dist": 30, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": "feral_humans_death_drops_photo",
+    "zombify_into": "mon_zombie_survivor",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "death_function": { "effect": { "id": "death_blind_oneineight", "hit_self": true } },
+    "upgrades": { "half_life": 45, "into": "mon_feral_human_photokin2" },
+    "special_attacks": [
+      {
+        "id": "psi_photokin1_lightdodge",
+        "type": "spell",
+        "spell_data": { "id": "photokinetic_light_dodge_monster", "min_level": 3 },
+        "cooldown": 10,
+        "monster_message": "%1$s's form is shifting with illusions!"
+      },
+      {
+        "id": "psi_photokin1_lightbeam",
+        "type": "spell",
+        "spell_data": { "id": "photokinetic_light_beam_monster", "min_level": 2 },
+        "cooldown": 15,
+        "monster_message": "%1$ fires a photon beam!"
+      }
+    ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "PATH_AVOID_DANGER_1",
+      "PRIORITIZE_TARGETS",
+      "PHOTOKIN_MONSTER_IMMUNE"
```

```suggestion
      "PRIORITIZE_TARGETS"
```
I'd say damage immunity is better saved for the higher tier psykers.